pkg/cmd/start/root.go

```diff
@@ -0,0 +1,145 @@
+	"github.com/nitrictech/cli/pkg/project"
+	"github.com/nitrictech/cli/pkg/run"
+	"github.com/nitrictech/cli/pkg/tasklet"
+)
+
+var envFile string
+
+var servicesCmd = &cobra.Command{
+	Use:         "start",
+	Short:       "Run nitric services locally for development and testing",
+	Long:        `Run nitric services locally for development and testing`,
+	Example:     `nitric start`,
+	Annotations: map[string]string{"commonCommand": "yes"},
+	Run: func(cmd *cobra.Command, args []string) {
+		term := make(chan os.Signal, 1)
+		signal.Notify(term, os.Interrupt, syscall.SIGTERM)
+		signal.Notify(term, os.Interrupt, syscall.SIGINT)
+
+		// Divert default log output to pterm debug
+		log.SetOutput(output.NewPtermWriter(pterm.Debug))
+
+		ls := run.NewLocalServices(&project.Project{
+			Name: "local",
+		})
+		if ls.Running() {
+			pterm.Error.Println("Only one instance of Nitric can be run locally at a time, please check that you have ended all other instances and try again")
+			os.Exit(2)
+		}
+
+		memerr := make(chan error)
+		pool := run.NewRunProcessPool()
+
+		startLocalServices := tasklet.Runner{
+			StartMsg: "Starting Local Services",
+			Runner: func(progress output.Progress) error {
+				go func(errch chan error) {
+					errch <- ls.Start(pool)
+				}(memerr)
+
+				for {
+					select {
+					case err := <-memerr:
+						// catch any early errors from Start()
+						if err != nil {
+							return err
+						}
+					default:
+					}
+					if ls.Running() {
+						break
+					}
+					progress.Busyf("Waiting for Local Services to be ready")
+					time.Sleep(time.Second)
+				}
+				return nil
+			},
+			StopMsg: "Started Local Services!",
+		}
+		tasklet.MustRun(startLocalServices, tasklet.Opts{
+			Signal: term,
+		})
+
+		pterm.DefaultBasicText.Println("Local running, use ctrl-C to stop")
+
+		stackState := run.NewStackState()
+
+		area, _ := pterm.DefaultArea.Start()
+		lck := sync.Mutex{}
+		// React to worker pool state and update services table
+		pool.Listen(func(we run.WorkerEvent) {
+			lck.Lock()
+			defer lck.Unlock()
+			// area.Clear()
+
+			stackState.UpdateFromWorkerEvent(we)
+
+			tables := []string{}
+			table, rows := stackState.ApiTable(9001)
+			if rows > 0 {
+				tables = append(tables, table)
+			}
+
+			table, rows = stackState.TopicTable(9001)
+			if rows > 0 {
+				tables = append(tables, table)
+			}
+
+			table, rows = stackState.SchedulesTable(9001)
+			if rows > 0 {
+				tables = append(tables, table)
+			}
+			area.Update(strings.Join(tables, "\n\n"))
+		})
+
+		select {
+		case membraneError := <-memerr:
+			fmt.Println(errors.WithMessage(membraneError, "membrane error, exiting"))
+		case <-term:
+			fmt.Println("Shutting down services - exiting")
+		}
+
+		_ = area.Stop()
+		// Stop the membrane
+		cobra.CheckErr(ls.Stop())
+	},
+	Args: cobra.ExactArgs(0),
+}
+
+func RootCommand() *cobra.Command {
+	servicesCmd.Flags().StringVarP(&envFile, "env-file", "e", "", "--env-file config/.my-env")
```

Comment:
why do you have this? it seems to not be supported..

Reply:
need to remove that

Reply:
fixed